mmcv/runner/hooks/logger/wandb.py

```diff
@@ -59,20 +60,26 @@ def before_run(self, runner):
             self.wandb.init()
         
         if self.config_path is not None:
-            shutil.copy2(self.config_path, self.wandb.run.dir)
             if os.path.isdir(self.config_path):
-                for path, _, _ in os.walk(self.wandb.run.dir):
+                copy_tree(self.config_path, self.wandb.run.dir)
```

Comment:
Is it necessary to copy the directory to `self.wandb.run.dir` first and then save (upload) the `self.wandb.run.dir` to wandb?